doc/examples/simple/gitlab/Makefile

```diff
@@ -36,7 +36,7 @@ publish:
 
 alerts.json:	sa
 	docker run --rm  -v ${CI_PROJECT_DIR}:/data -w /data  ${REDEMPTION_IMAGE}  python3 /host/code/analysis/alerts2input.py  src  cppcheck_oss  cppcheck.xml  alerts.json
-
+#[NOTE] --alerts isn't used here. Only path of /data/alerts.json, which conflicts with the documentation in /redemption/doc/examples/simple/README.MD in Execution
```

Comment:
I will update the Makefiles to use --alerts (after confirming that it still works).